model/bin/ww3_from_ftp.sh

```diff
@@ -57,6 +57,8 @@ cp -r data_regtests/mww3_test_09/input/*   regtests/mww3_test_09/input/
 cp -r data_regtests/ww3_ufs1.1/input/*     regtests/ww3_ufs1.1/input/
 cp -r data_regtests/ww3_ufs1.1/input/*.nc  regtests/ww3_ufs1.2/input/
 cp -r data_regtests/ww3_ufs1.2/input/*     regtests/ww3_ufs1.2/input/
+cp -r data_regtests/ww3_ufs1.1/input/glo_15mxt*  regtests/ww3_ufs1.3/input/
```

Comment:
I believe this is a deviation from operations, want to confirm that's a known choice being made.

Reply:
I removed this and used the one from workflow